src/app/(sidebar)/transaction/build/components/TransactionXdr.tsx

```diff
@@ -101,31 +101,57 @@ export const TransactionXdr = () => {
           case "starting_balance":
             return xdrUtils.toAmount(val);
           // Number
+          case "bump_to":
           case "offer_id":
             return BigInt(val);
           // Price
           case "price":
             return xdrUtils.toPrice(val);
+          case "data_value":
+            return Buffer.from(val).toString("hex");
+          case "balance_id":
+            return {
+              claimable_balance_id_type_v0: val.replace(/^(00000000)/, ""),
```

Comment:
is it safe to remove `00000000`? I saw your question on slack but I didn't see a response